remotepy/instance.py

```diff
@@ -571,5 +571,72 @@ def launch(
     )
 
 
+@app.command()
+def terminate(instance_name: str = typer.Argument(None, help="Instance name")):
```

Comment:
Is this related to this PR?

Reply:
no, that's weird. That Pr was merged to main